prereise/gather/griddata/hifld/data_access/load.py

```diff
@@ -302,9 +302,12 @@ def get_hifld_electric_power_transmission_lines(path):
     properties = (
         pd.DataFrame([line["properties"] for line in data["features"]])
         .astype({"ID": "int64", "NAICS_CODE": "int64", "SOURCEDATE": "datetime64"})
-        .drop(columns=["OBJECTID", "SHAPE_Length"])
         .round({"VOLTAGE": 3})
     )
+    # Drop columns which aren't relevant to us. Some names are inconsistent b/w versions
+    shape_lengths = {"SHAPE_Length", "SHAPE__Length"}
```

Comment:
That is bad

Reply:
Indeed. And this looks familiar to me which reminds me the days playing with country names from different datasets.